src/IODD.Provider/IODDFinderPublicClient.cs

```diff
@@ -10,7 +11,15 @@ public class IODDFinderPublicClient : IIODDProvider
 
     public IODDFinderPublicClient(Uri baseUrl)
     {
-        _httpClient = new() { BaseAddress = baseUrl };
+        var handler = new HttpClientHandler
```

Comment:
Let us make this optional and configurable via ctor parameter. I think it should be up to the user whether validation should be in place or not.